lib/little-loader.js

```diff
@@ -51,27 +55,30 @@
    * @param {Object}    context   (Optional) Callback context
    * @returns {void}
    */
-  window._lload = function (src, callback, context) {
+  var _lload = function (src, callback, context) {
     /*eslint max-statements: [2, 20]*/
     context = context || this;
```

Comment:
Even though this is a known safe operation there is nothing that says we won't refactor this and expose the method at some time in the future. Leaving this as is means that future you has to remember to correct the argument mutation. Also, you can't wear the badge of "functionally pure" if you're doing mutation like that :).

@exogen - use strict makes largeish change to how JS runs but it won't correct anything here. It is highly advised that you use it but that seems like a separate issue and possibly a separate ticket.